ivy/functional/frontends/numpy/logic/truth_value_testing.py

```diff
@@ -57,3 +57,22 @@ def isscalar(element):
 @to_ivy_arrays_and_back
 def isfortran(a: np.ndarray):
     return a.flags.fnc
+
```

Comment:
There are some link errors:
```
ivy/functional/frontends/numpy/logic/truth_value_testing.py:64:89: E501 line too long (92 > 88 characters)
ivy/functional/frontends/numpy/logic/truth_value_testing.py:78:1: W391 blank line at end of file
```
Did you installed the [pre-commit hook](https://lets-unify.ai/ivy/contributing/setting_up.html#pre-commit)?